src/domain/checklist/preLoadChecklist.ts

```diff
@@ -5,6 +5,18 @@ import * as O from 'fp-ts/Option';
 import { PreLoadRecord } from '../PreLoadRepository';
 import { Checklist } from './types';
 
+// Another way to do this is to split the server part that records
```

Comment:
Another option is to write a simple express middleware that records request and response.